articles/libraries/chemistry/concepts/second-quantization.md

```diff
@@ -252,8 +250,13 @@ In particular, if $\psi\_j$ are the spin orbitals that form the basis then
 where $h\_{\textrm nuc}$ is the nuclear energy (which is a constant under the Born-Oppenheimer approximation) and
 
 \begin{align}
-h\_{pq} &= \int\_{-\infty}^\infty \psi^\*\_p(x\_1) \left(\sum\_{i} \frac{|\hat{p}\_i|^2}{2m\_e} -\sum\_{i, k} \frac{e^2}{|\hat{x}\_i - y\_k|}\right)  \psi\_q(x\_1)\mathrm{d}^3x\_1\nonumber\\\
-h\_{pqrs} &= \int\_{-\infty}^\infty \int\_{-\infty}^\infty\psi\_p^\*(x\_1)\psi\_q^\*(x\_2) \left(\frac{1}{2}\sum\_{i\ne j} \frac{e^2}{|\hat{x}\_i - \hat{x}\_j|} \right)\psi\_r(x\_2)\psi\_s(x\_1)\mathrm{d}^3x\_1\mathrm{d}^3x\_2.\tag{★}\label{eq:integrals}
+h\_{pq} &= \int\_{-\infty}^\infty \psi^\*\_p(x\_1) \left(-\frac{\nabla^2}{2} +V(x\_1)\right)  \psi\_q(x\_1)\mathrm{d}^3x\_1,
+\end{align}
+
+where $V(x)$ is the mean-field potential, and
+
+\begin{align}
+h\_{pqrs} &= \int\_{-\infty}^\infty \int\_{-\infty}^\infty\psi\_p^\*(x\_1)\psi\_q^\*(x\_2) \left(\frac{1}{|x_1-x_2|} \right)\psi\_r(x\_2)\psi\_s(x\_1)\mathrm{d}^3x\_1\mathrm{d}^3x\_2.\tag{★}\label{eq:integrals}
```

Comment:
Perhaps this was already the case, sorry, but in the rendering it looks like now two equations have the tag (★)?
![image](https://user-images.githubusercontent.com/31516/59639862-3fa8a800-9111-11e9-8884-02ae1f68a64b.png)

Reply:
Removed the ★ tag